geth/api/backend_test.go

```diff
@@ -206,7 +206,7 @@ func (s *BackendTestSuite) TestCallRPC() {
 				"data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}],"id":1}`,
 			func(resultJSON string) {
 				log.Info("eth_sendTransaction")
-				s.T().Log("GOT: ", resultJSON)
+				s.NotContains(resultJSON, "error")
```

Comment:
Does it make sense to check for exact response here? `require.Equal(...)` for example.

Reply:
We don't know the exact response as it contains hash which is pretty random.